concent_api/common/decorators.py

```diff
@@ -174,7 +175,9 @@ def wrapper(request, client_message, client_public_key, *args, **kwargs):
                 response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
                 if database_name is not None:
                     transaction.savepoint_commit(sid, using=database_name)
-            except (Http400, FileTransferTokenError, HashingAlgorithmError, GolemMessageValidationError) as exception:
+            except (
+                Http400, FileTransferTokenError, HashingAlgorithmError, GolemMessageValidationError, ConcentValidationError
```

Comment:
We need to make decision if we allow `ConcentValidationError` to be caught here instead of view first. @dybi @cameel What do you guys think ? I have added same question but more detailed in https://github.com/golemfactory/concent/pull/608#pullrequestreview-134832811 .

Reply:
I think we can even catch `ConcentBaseException` - it will be more succinct

Reply:
Depends on what inherits from `ConcentBaseException`. If there are any errors that indicate failures, they are not supposed to get a HTTP 4xx response.